tests/foreman/conftest.py

```diff
@@ -183,4 +183,5 @@ def pytest_collection_modifyitems(items, config):
 
 @pytest.fixture(autouse=True, scope="function")
 def record_test_timestamp_xml(record_property):
-    record_property("start_time", int(time() * 1000))
+    now = datetime.datetime.now()
```

Comment:
What about `.utcnow()` - that should return datetime object with TZ set to UTC?

https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

But feel free to ignore me as I admitt this timezone business is quite confusing to me.